lib/helpers.bash

```diff
@@ -654,19 +657,15 @@ function _bash-it-restart() {
 	_about 'restarts the shell in order to fully reload it'
 	_group 'lib'
 
-	local saved_pwd="${PWD}" init_file="${BASH_IT_BASHRC:-${HOME?}/.bashrc}"
```

Comment:
why remove this one?
this way, we wont `cd` back into the original directory

Reply:
I removed all the `cd`ing around. It just stays in the original directory; no need to change out or back.